pom.xml

```diff
@@ -31,12 +31,19 @@
     <jmhjar.name>benchmarks</jmhjar.name>
     <tycho-version>1.2.0</tycho-version>
     <cbi-plugins.version>1.1.5</cbi-plugins.version>
-    <surefire.version>2.22.0</surefire.version>
     <junit.version>5.3.0</junit.version>
-    <maven-plugin-tools.version>3.5.2</maven-plugin-tools.version>
     <maven.version>3.5.0</maven.version>
     <unix.socket.tmp></unix.socket.tmp>
 
+    <!-- some maven plugins versions -->
+    <surefire.version>2.22.0</surefire.version>
+    <maven.compiler.plugin.version>3.8.0</maven.compiler.plugin.version>
+    <maven.dependency.plugin.version>3.1.1</maven.dependency.plugin.version>
+    <maven.resources.plugin.version>3.1.0</maven.resources.plugin.version>
+    <maven.source.plugin.version>3.0.1</maven.source.plugin.version>
+    <maven.war.plugin.version>3.2.2</maven.war.plugin.version>
+    <maven-plugin-tools.version>3.5.2</maven-plugin-tools.version>
```

Comment:
Ditto, same format as others here?

Reply:
I actually prefer the format `maven.plugin.xxx.version` so that it's immediate that they are maven plugin related properties.

Reply:
sounds a nice change